documentation/README.md

```diff
@@ -34,6 +41,34 @@ a pull request if you do something interesting.
 1. Lastly, you will start-up the containers.  This can be done from the terminal/command line by navigating to the folder
    containing the `docker-compose.yml` file and running a docker-compose up command.
 
+## Configuration on Mac OS X and Linux Docker hosts
```

Comment:
@pbchase - are you trying to say that people shouldn't modify the .env file at all and instead should manually make an override file that sets environmental vars?    I like that change!  Maybe we should edit the .env file so at hte top is tells people not to modify this file directly (unless you really want to)...